pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -2,10 +2,11 @@ package unifiedpushserver
 
 import (
 	"context"
-	"k8s.io/apimachinery/pkg/api/resource"
 	"reflect"
 	"time"
 
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
There are other packages from k8s.io/apimachinery down around line 27 -- maybe this could be put down there?